src/osbuild-manifests/platform.live-iso.ipp.yaml

```diff
@@ -0,0 +1,73 @@
+# This file defines the pipeline for building the live ISO.
+version: '2'
+pipelines:
+  - name: live-iso
```

Comment:
since we are copying in the metal.raw and metal4k.raw images here when we copy out of this pipeline it will include those metal.raw and metal4k.raw images which isn't what we want.

what we'll have to do is have one stage for creating the live artifacts and another stage where we copy only the created artifacts, which then gets copied out.

That is... unless we figure out how to do the loopback device stuff inside the stage. If we can do the loopback stuff inside the stage code then we can use an `input:` instead from a previous pipeline instead and then set up the loopback device from within the stage code.

I guess another option would be to teach the `devices/org.osbuild.loopback` code to somehow work on inputs.. but I'm not sure if it can be done easily or if it would be accepted.